### ISSUE TITLE
Projects using docz 🙌🏻

### ISSUE THREAD

**Issue by pedronauck** [doczjs/docz#37]
## Question

Hey guys, I'm really impressed with the number of people asking me about how use docz and this gave me an idea... I'm thinking about creating a section on [docz.site](https://docz.site) listing all project that is using Docz as documentation tools, please give me some links 🙏 

What do you thinking about that? 😍

**Comment by renatorib** [comment]
Powerplug https://github.com/renatorib/react-powerplug will have a docs soon. :)

**Comment by LightDiscord** [comment]
[islands-wars/Docs](https://github.com/islands-wars/Docs) We have migrated from Gitbook to Docz, although there are still some small problems.

**Comment by EdwardIrby** [comment]
I'm thinking of doing Moovel's design system in it and won't mind sharing once I get the approval.

**Comment by fzaninotto** [comment]
I'm trying docz for [react-admin](https://github.com/marmelab/react-admin/pull/1939), but so far the basic docz features (PropTable, Playground) show limits that I can't overcome.

**Comment by fzaninotto** [comment]
Most of them have already been identified and filed as issues. See https://github.com/marmelab/react-admin/pull/1939#issuecomment-398981272 for details.

**Comment by felipeblini** [comment]
Congratulations guys for the awesome project! Its work only for React?

**Comment by fzaninotto** [comment]
If your question is "Did we also develop a Vue.js version", the answer is no.

**Comment by pedronauck** [comment]
@felipeblini we're working now in a integration with Vue, Preact and Inferno 👍

**Comment by tino** [comment]
@pedronauck please add a note on the site about the “things” docz can document, because I had to come to this ticket to find out :)

**Comment by drmzio** [comment]
Just deployed https://github.com/ompmega/react-uikit and I'm loving it!

**Comment by pedronauck** [comment]
Some projects using docz:

- https://github.com/koddr/react-goodshare-components
- https://github.com/istarkov/rifm

**Comment by inlet** [comment]
We’ve just created our docz for this awesome project https://reactpixi.org

Love it, but unfortunately the default theme isn’t working properly on mobile. Hope this will be fixed soon. 

Keep up the good work!

**Comment by pedronauck** [comment]
Nice @inlet, really good  👏👏 

So, what's happening on mobile that we need to fix to be able to your work? Please, tell me, maybe if is something quick I can fix for you 🙏

**Comment by inlet** [comment]
Hi @pedronauck,

Here are my findings (tested it with iOS 11.2):

https://paper.dropbox.com/doc/Docz-Mobile-Improvements--AHZKJfQdy~Q4l1Ua3kPZsf8hAQ-UGw8HdfU2duCWlzEJ28y4

It already did save me a lot of time and love the fact that it's super easy to use and to extend.  Would love to see some improvements on the default theme.

Thanks!

**Comment by PanJiaChen** [comment]
@pedronauck 

I really like this project, but my project is based on `vue`. How it is going now?

**Comment by Istom1n** [comment]
Hi @pedronauck, it's a good idea, but I think better to add projects with own design, as a minimum.

I tried to use docz for blogging, but have a problem with katex render.

**Comment by ronalson** [comment]
I'm looking forward for the Vue update (#95)! :)

**Comment by EmaSuriano** [comment]
Hello guys! I rewrite the documentation of [react-hotkey-tooltip](https://github.com/EmaSuriano/react-hotkey-tooltip) with Docz, this is the new [documentation](https://react-hotkey-tooltip.netlify.com/#/) 😄 You can use it as an example if you want!

**Comment by benhinchley** [comment]
[@sajari/sdk-react](https://www.npmjs.com/package/@sajari/sdk-react) uses docz for our [documentation](https://sajari-sdk-react.netlify.com/).

**Comment by ilyanoskov** [comment]
@pedronauck we are finally using docz for Circuit Ui at SumUp! We just finished setting everything up, and are excited to have our own component documentation!

https://sumup.github.io/circuit-ui

Thank you so much for your amazing work, this project is 🔥

**Comment by rakannimer** [comment]
Hey @pedronauck,

Great work on docz ! 
I just migrated [react-google-charts docs](https://rakannimer.github.io/react-google-charts/#/) to it. 

[Docs code.](https://github.com/rakannimer/react-google-charts/tree/master/src/docs)

**Comment by liaoanran** [comment]
@pedronauck  hi，I'm looking forward for the Vue update，when to support it?

**Comment by Rhuansantos** [comment]
Florida Blue / Guidewell is using Docz in many different internal projects!

**Comment by ritz078** [comment]
We have started with docz for our design system. Since it was a pain setting up storybook and babel to play well together, I set up a boilerplate for others to use. https://github.com/ritz078/react-typescript-component-starter-kit

**Comment by santosfrancisco** [comment]
Docz is really amazing! I use it for the lib react-awesome-styled-grid documentation. https://awesome-styled-grid.netlify.com

tks!

**Comment by yo7** [comment]
Smooth-UI is using docz👀 (I’m not involved with the project though)
https://smooth-ui.smooth-code.com/

**Comment by capaj** [comment]
I use it for my GraphQL schema generator project https://decapi.netlify.com/. Even though I've hit an issue with the netlify guide- #409 and my anchor tag links are not working for some reason. Otherwise very nice so far!

**Comment by michelson** [comment]
A very rudimentary layout implementation at https://michelson.github.io/dante2 . Just waiting for [this](http://roadmap.docz.site/roadmap/p/new-package-docz-ui) to enhance the playground
awesome project, thanks!

**Comment by rot1024** [comment]
Hi, I'm using docz with resium.

Documentation: http://resium.darwineducation.com
GitHub: https://github.com/darwin-education/resium

Thank you for your nice library!

**Comment by adeelibr** [comment]
Can I make a PR to add my OSS library built on docz as well, I am more then happy to make a PR for it https://melting-pot.netlify.com/ This is fairly new.

**Comment by drcmda** [comment]
We use it for react-spring: http://react-spring.surge.sh/platforms and https://github.com/react-spring/react-spring :)

**Comment by sw-yx** [comment]
## list of everyone so far

---

Libraries

- react-powerplug https://github.com/renatorib/react-powerplug
- react-uikit https://github.com/ompmega/react-uikit
- react-goodshare-components https://github.com/koddr/react-goodshare-components
- react input format & mask https://github.com/istarkov/rifm
- react-pixijs https://reactpixi.org/
- react-hotkey-tooltip https://react-hotkey-tooltip.netlify.com/#/
- sajari react sdk (search library) https://sajari-sdk-react.netlify.com/
- react-google-charts https://react-google-charts.com/ https://github.com/rakannimer/react-google-charts/tree/master/src/docs
- brookjs https://valtech-nyc.github.io/brookjs/
- react-typescript-component-starter-kit https://github.com/ritz078/react-typescript-component-starter-kit
- awesome-styled-grid https://awesome-styled-grid.netlify.com/
- decapi (graphql schema generator) https://decapi.netlify.com/
- dante2 https://michelson.github.io/dante2
- melting pot https://melting-pot.netlify.com/
- react-spring http://react-spring.surge.sh/

Companies

- circuit UI (SumUP design system) https://sumup.github.io/circuit-ui
- smooth UI https://smooth-ui.smooth-code.com/
- resium/darwin education http://resium.darwineducation.com/

**Comment by rot1024** [comment]
> - resium/darwin education http://resium.darwineducation.com/

Please fix as bellow:

Product name: Resium
URL: https://resium.darwineducation.com/
Company name is `Darwin Education, Inc.

Thanks!

**Comment by alanrsoares** [comment]
Name: re-reduced
URL: https://re-reduced.netlify.com/
Github: https://github.com/alanrsoares/re-reduced

**Comment by aryzing** [comment]
We published our first iteration yesterday 🎉 

Company name: Ninety Percent of Everything ([logo link](https://avatars1.githubusercontent.com/u/35486389?s=200&v=4))
docz URL: https://90poe.github.io/
Github: https://github.com/90poe
Company site: https://90poe.io

**Comment by rfoel** [comment]
Just deployed https://rebasejs.com 🎉

**Comment by jedmao** [comment]
# [![Queso](https://github.com/jedmao/queso/raw/master/media/header.png)](https://github.com/jedmao/queso)

Docz Site: https://queso.surge.sh/
GitHub: https://github.com/jedmao/queso

**Comment by cdaringe** [comment]
https://cowtown.surge.sh

**Comment by cdaringe** [comment]
@jedmao--nice, I was just about to start looking into a `runkit` integration, and I see you've already done so.  I intend to emulate your solution. thx for the ref!

**Comment by jedmao** [comment]
@cdaringe RunKit is actually throwing errors so I'm trying to switch over to a Monaco editor.

**Comment by ivan-dalmet** [comment]
[![Saagie Design System](https://7-design-system.public.prod.saagie.io/v/latest/public/images/docs-logo.svg)](http://go.saagie.com/design-system)

🎉 We just publish a new version with **Docz 1.0**

**URL:** [go.saagie.com/design-system](go.saagie.com/design-system)
**Company**: Saagie

_Thanks for your work @pedronauck (and everyone helping to build Docz 💪)_

**Comment by lemon-31** [comment]
Ultra Config are now proudly using Docz for our documentation:

https://ultraconfig.com.au/ultra-config-generator/docs/

**Comment by adeelibr** [comment]
@lemon-31 oh wow! how did you end up integrating this with your entire website? I see their is a top horizontal menu. I like it.

**Comment by FredyC** [comment]
MobX in React site made with the help of Docz ❤️ 

https://mobx-react.js.org/

**Comment by lemon-31** [comment]
Hello @adeelibr ,

The website is built using GatsbyJS. There is a tutorial on how to integrate Docz into an existing Gatsby site. Link: https://www.gatsbyjs.org/docs/writing-documentation-with-docz/

For the header and footer you can use the "wrapper" option inside the doczrc.js file. Also, some tweaking to the CSS is needed to make space for the header and footer.

**Comment by xxczaki** [comment]
PizzaQL Docs are built with Docz :smile:

https://docs.pizzaql.tech

**Comment by yyynnn** [comment]
Hi!

**MTS Bank** is using and **MTS** will be using **Docz** soon for their own uikit documentation!

Check it out [https://yyynnn.github.io/mtsbankui](https://yyynnn.github.io/mtsbankui)

**Comment by fraincs** [comment]
> Hi!
> 
> **MTS Bank** is using and **MTS** will be using **Docz** soon for their own uikit documentation!
> 
> Check it out https://yyynnn.github.io/mtsbankui

I suggest holding your breath, or at least test before starting using it, development is stale.

**Comment by yyynnn** [comment]
> > Hi!
> > **MTS Bank** is using and **MTS** will be using **Docz** soon for their own uikit documentation!
> > Check it out https://yyynnn.github.io/mtsbankui
> 
> I suggest holding your breath, or at least test before starting using it, development is stale.

Yeah, WIP (as much as docz is, lol), despite the fact it is making 💵💵💵 already.

**Comment by pedronauck** [comment]
Just to point you guys, Docz now will be alive again, @rakannimer took the project maintaining along with me and is already making a very good job with the stuck issues/prs 🙌

**Comment by pepibumur** [comment]
I"m using it for Tuist's documentation too. Here's the website
https://docs.tuist.io

**Comment by cardner** [comment]
Are we able to post our sites here if it's not publically available? (Screenshots and such?)

**Comment by rakannimer** [comment]
@cardner yeah for sure ! Share what you can :)

**Comment by cardner** [comment]
Thanks @rakannimer 

**Duo Security Design System: Tellaro**
**url:** [https://tellaro.design](https://tellaro.design/) (in case we ever make it public)

**Screens:**
![tellaro design_ (1)](https://user-images.githubusercontent.com/4907876/67047171-c4c3be00-f0ff-11e9-89ea-9ca1d952f233.png)
![tellaro design_ (2)](https://user-images.githubusercontent.com/4907876/67047185-d016e980-f0ff-11e9-9b2e-1bf04c5abe83.png)
![tellaro design_ (3)](https://user-images.githubusercontent.com/4907876/67047199-d7d68e00-f0ff-11e9-97d8-a5144113181f.png)
![tellaro design_ (4)](https://user-images.githubusercontent.com/4907876/67047202-da38e800-f0ff-11e9-890b-66d4fa37f674.png)
![tellaro design_ (5)](https://user-images.githubusercontent.com/4907876/67047210-df963280-f0ff-11e9-9ebc-a7d475ab06da.png)
![tellaro design_ (6)](https://user-images.githubusercontent.com/4907876/67047214-e15ff600-f0ff-11e9-9685-87a43cf7ac1d.png)
![tellaro design_ (7)](https://user-images.githubusercontent.com/4907876/67047222-e3c25000-f0ff-11e9-8151-b4c91a4f11cc.png)
![tellaro design_ (8)](https://user-images.githubusercontent.com/4907876/67047225-e58c1380-f0ff-11e9-91fd-e76fb86023e9.png)

**Comment by AryanJ-NYC** [comment]
https://www.reactandtypescript.dev/

**Comment by PaulieScanlon** [comment]
I've just started using the gatsby theme version of docz for [gatsby-mdx-embed](https://gatsby-mdx-embed.netlify.com/

Great project by the way!

**Comment by pedronauck** [comment]
Such a great project made with Docz 😍 
http://welcome-ui.com/

**Comment by alexandre-lelain** [comment]
**Big thanks** to the contributors of this project!

I just re-wrote my react library documentation site from *Storybook.js* to *Docz*: it offers so much freedom and its built-in components are really awesome. Not to mention the huge gain in bundle size reduction thanks to Gatsby.js :)

Here is the link if you want to check it out: https://components-extra.netlify.com

**Comment by pedronauck** [comment]
Awesome @alexandre-lelain, glad you liked Docz! 🙏

**Comment by theo-mesnil** [comment]
Docz ❤️ run on http://welcome-ui.com

Thanks to the contributors  🎉 

Project source : https://github.com/WTTJ/welcome-ui/

**Comment by pedronauck** [comment]
Amazing know about this @alexandre-lelain, glad you liked and are using Docz 🙏

**Comment by ParamagicDev** [comment]
Still in its infancy, however I figured I'd share what I have so far. Kind of unconventional, but I'm rewriting my Paramedic state protocols from a PDF to a searchable HTML format.

[https://inspiring-varahamihira-efb922.netlify.com/](https://inspiring-varahamihira-efb922.netlify.com/)

[PDF for reference](https://www.mass.gov/doc/emergency-medical-services-pre-hospital-treatment-protocols-version-20201-effective-4120-pdf/download)
